docs/dev-setup.md

```diff
@@ -174,6 +174,32 @@ refer to
 [the development setup guide of CAPM3](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/dev-setup.md#tilt-for-dev-in-capm3)
 and specially the [Baremetal Operator Integration](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/dev-setup.md#including-baremetal-operator-and-ip-address-manager)
 
+### Making (virtual) BareMetalHosts with Tilt interface
+
+After Tilt is up it is possible to make baremetalhosts by pressing a button in the Tilt localhost interface (right upper corner). This button runs the content of file
+
+```sh
+tools/bmh_test/create_bmh.sh <NAME> <VBMC_PORT>
+```
+
+and adds the values given to the button as arguments. Controlplane host can be created with
+
+```sh
+tools/bmh_test/create_bmh.sh <NAME> <VBMC_PORT> <CONSUMER> <CONSUMER_NAMESPACE>
+```
+
+[Virtualbmc](https://pypi.org/project/virtualbmc/) needs to be downloaded to make this work. The network and VBMC needed for making a baremetalhost can be initialized with
+
+```sh
+sudo tools/bmh_test/run_local_bmh_test_setup.sh
+```
+
+The network, VBMC, virtual vbmc environment and virtual machines can be run down with
```

Comment:
imo "clean up" might be a better wording than "run down", at least I wouldn't understand that term.